tests/integration/test_unambiguous_alter_commands/configs/format_alter_operations_with_parentheses.xml

```diff
@@ -0,0 +1,3 @@
+<clickhouse>
+    <format_alter_operations_with_parentheses>1</format_alter_operations_with_parentheses>
```

Comment:
We can enable it in functional tests by adding this config to `tests/config/install.sh`
Btw, functional tests are much better than integration tests: https://github.com/ClickHouse/ClickHouse/pull/39359#issuecomment-1188296170
(but it's not necessary to rewrite existing tests)

Reply:
I checked this, and if we would change this in functional tests, then we would change this for all of the functional tests. I think it shouldn't cause an issue, but we would test something that is not the "default" behavior of clickhouse.

Furthermore I used the same queries in functional tests, but expecting the old behavior. So I would say the only question is which way should be the default in functional tests? The other one has to be tested in integration test either way, because we have to change the server setting.